src/core/meta.ml

```diff
@@ -44,7 +44,7 @@ let get_documentation d =
 		) flags;
 		let params = (match List.rev !params with
 			| [] -> ""
-			| l -> "(" ^ String.concat "," l ^ ")"
+			| l -> "(<" ^ String.concat ">, <" l ^ ">) "
```

Comment:
Why change the formatting of params?

Reply:
Thought it would be less ambiguous if each parameter is in `<...>`, since `"params"` contains multi-word descriptions.